spec/dummy/config/routes.rb

```diff
@@ -1,2 +1,10 @@
 Rails.application.routes.draw do
+  namespace :rails do
```

Comment:
Maybe after migration to engine we can remove it?

Reply:
We can`t